InternProject/DAL/Models/DeliveryTrack/DCBookingDetailModel.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using InternProject.DAL.Models.OrderTrack;
+using InternProject.DAL.Models.ShipmentTrack;
+
+namespace InternProject.DAL.Models.DeliveryTrack
+{
+    public class DCBookingDetailModel
+    {
+        [Key]
+        public int Id { get; set; }
```

Comment:
Create a base model for all entity to include all common fields such as Id

Reply:
change field names to match naming convention ("ID" to "Id")